lib/ruby_warrior/abilities/talk.rb

```diff
@@ -0,0 +1,24 @@
+module RubyWarrior
+  module Abilities
+    class Talk < Base
+      def description
+        "Initiates a conversation between the warrior and an informant."
+      end
+
+      def perform(direction = :forward)
+        verify_direction(direction)
+        receiver = unit(direction)
+        if receiver
+          @unit.say "faces #{direction} and talks to #{receiver}"
+          if receiver.character == "i"
```

Comment:
Can't we leverage the receiver class type here, instead of the character?
That seems too tightly coupled

Reply:
Good point